docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-helm-chart-5.x-to-6.0.md

```diff
@@ -0,0 +1,84 @@
+---
+description: "Migrate the Helm chart from version 5.x to 6.0"
+title: "Migrate the Helm chart from version 5.x to 6.0"
+menuTitle: "Migrate Helm chart 5.x to 6.0"
+weight: 120
+---
+
+# Migrate the Helm chart from version 5.x to 6.0
+
+The `mimir-distributed` Helm chart version 6.0 introduces several breaking changes:
+
+- Ingest storage and Kafka are enabled by default. The bundled Kafka deployment is for demo and testing purposes only and is not suitable for production use.
+- Rollout-operator webhooks are enabled by default, which requires installing CRDs before upgrading.
+- The top-level `nginx` values section has been removed; you must migrate to the unified gateway deployment before upgrading to version 6.0.
+
+## Prerequisites
+
+- You are running `mimir-distributed` Helm chart version 5.x.
+- If using the rollout-operator, you have cluster permissions to install CRDs.
+- If you use the top-level `nginx` values, you have already migrated to the unified gateway deployment.
+
+## Procedure
+
+### 1. Migrate to unified gateway deployment (if needed)
+
+If your values file contains a top-level `nginx` section, you must migrate to the unified gateway deployment before upgrading to version 6.0.
+
+Follow the [Migrate to unified proxy deployment](https://grafana.com/docs/helm-charts/mimir-distributed/v5.8.x/migration-guides/migrate-to-unified-proxy-deployment/) guide to complete this migration.
+
+### 2. Install rollout-operator CRDs (if using rollout-operator)
+
+If you use the rollout-operator, install the CRDs from the rollout-operator chart:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/grafana/helm-charts/main/charts/rollout-operator/crds/replica-templates-custom-resource-definition.yaml
+kubectl apply -f https://raw.githubusercontent.com/grafana/helm-charts/main/charts/rollout-operator/crds/zone-aware-pod-disruption-budget-custom-resource-definition.yaml
+```
+
+### 3. Choose your ingest storage strategy
+
+Choose one of the following options:
+
+#### Option A: Keep using classic architecture (disable ingest storage)
+
+Classic architecture will be supported until Mimir 4.0 and removed in 5.0. If you want to continue using the classic architecture without ingest storage, add the following to your values file:
```

Comment:
I didn't realize we'd planned out the versions classic architecture would be removed in. This sounds reasonable to me but maybe hedge the language here and talk to the team about it?


```suggestion
Classic architecture will be supported until at least Mimir 4.0. If you want to continue using the classic architecture without ingest storage, add the following to your values file:
```